CHANGELOG.md

```diff
@@ -8,11 +8,15 @@
 * Remove now-unused `config.IsDevelopmentEnv()` helper to encourage better
   testing practices. Not a breaking change as nobody is using this func
   themselves according to our code search tool.
-* Log `traceID` and `spanID` in hex format to match Jaeger UI. Upgrade Jaeger to min version 2.1.0
+* Log `traceID` and `spanID` in hex format to match Jaeger UI. Upgrade Jaeger to
```

Comment:
Sorry, this was breaking markdownlint which runs as part of overcommit, which was annoying because I had to `git commit --no-verify` every time I updated the changelog